### DISCUSSION TITLE
How to play multiple audio tracks simultaneously with lossless cut?

### DISCUSSION THREAD

**Discussion by Avnsx** [mifi/lossless-cut#1388] [Support]
Hey, my bad you most likely [answered this](https://github.com/mifi/lossless-cut/issues/256#issuecomment-615834864) many times before, but I kinda didn't understand all the stuff I read about the topic of playing multiple audio tracks simultaneously?

As far as I see I can only listen to a single track at once, right?

I've mp4 footage with 5 audio tracks and I really need to listen to atleast two of them. I can see them visually in "Tracks" but that's about it, it won't allow me to listen to them at the same time.

![sample](https://i.imgur.com/fgC9Bme.png)

Also I've an idea, if the html 5 player doesn't allow to play multiple tracks at once, why don't you make a simple menu or something where people can just combine the tracks they want to listen to on a single track (for editing purposes for example). So then lossless cut just merges those tracks together and plays them as one until it gets exported?

**Comment by mifi** [comment]
Hi. Yes currently it's only possible to listen to the "main" audio track unfortunately. but rightly that issue handles a feature request to allow this.

**Reply by mifi** [reply]
It’s also been 600 other new issues submitted the last 3 years and I only have so much time in my life. Luckily for you this particular feature has been implemented. See #256
You can try it in the nightly build
